rc/base/file.kak

```diff
@@ -0,0 +1,13 @@
+decl str mimetype
+
+hook global BufOpen .* %{ %sh{
+    if [ -z "${kak_opt_filetype}" ]; then
+        mime=$(file -b --mime-type "${kak_buffile}")
+        printf %s\\n "${mime}" | grep -q '^text/x-'
```

Comment:
Not all mimetypes are easily linkable to their respective filetype, e.g. `message/rfc822` for mails. This list is going to be big if it's implemented, and not having it would allow weird mimetypes to be contained in their respective language scripts.